### PR TITLE
sstp: T4393: Add support to configure host-name (SNI)

### PR DESCRIPTION
**Pull request by natali-rs1985**
Add option to vpn sstp ```set vpn sstp host-name <hostname>```
## Related Task(s)

* https://vyos.dev/T4393

## Related PR(s)

https://github.com/vyos/vyos-1x/pull/3436
## Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-documentation/blob/current/CONTRIBUTING.md) document

### CODE REVIEW

File: docs/configuration/vpn/sstp.rst [hunk 1]
```diff
@@ -471,6 +471,10 @@ Global Advanced options
 
   Windows Internet Name Service (WINS) servers propagated to client
 
+.. cfgcmd:: set vpn sstp host-name <hostname>
+
+  Only allow connection to specified host with the same TLS SNI
```
Comment (c-po) [inline docs/configuration/vpn/sstp.rst]: Can we use the sentence from MAN page: `If this option is given, only sstp connection to specified host and with the same TLS SNI will be allowed.`

**Review by c-po — CHANGES_REQUESTED** [review]

**Comment by c-po** [comment]
@MergifyIo backport sagitta

File: docs/configuration/vpn/sstp.rst (hunk 1 above)
Comment (natali-rs1985) [reply docs/configuration/vpn/sstp.rst]: Changed

File: docs/configuration/vpn/sstp.rst [hunk 2]
```diff
@@ -471,6 +471,11 @@ Global Advanced options
 
   Windows Internet Name Service (WINS) servers propagated to client
 
+.. cfgcmd:: set vpn sstp host-name <hostname>
+
+  If this option is given, only sstp connection to specified host
```
Comment (dmbaturin) [inline docs/configuration/vpn/sstp.rst]: "SSTP" is an abbreviation and must be capitalized.

**Review by dmbaturin — CHANGES_REQUESTED** [review]

**Pull request merged**